nautobot/docs/release-notes/version-1.0.md

```diff
@@ -4,180 +4,10 @@ This document describes all new features and changes in Nautobot 1.0, a divergen
 
 Users migrating from NetBox to Nautobot should also refer to the ["Migrating from NetBox"](../installation/migrating-from-netbox.md) documentation as well.
 
-## v1.0.0b4 (2021-04-19)
```

Comment:
Personally I'm not in favor of merging all the per-beta listings of added/changed/fixed/removed items into one consolidated "1.0.0" listing - it feels a little bit like rewriting history. My vote (others can certainly disagree) would be to have the document keep its existing structure and just add the high-level summary of major changes at the top of the document.

Reply:
Yeah I agree with that. IMO the intent of having a unified release notes is really just about summarizing the changes with descriptions like we did in the initial b1 release, but having that be something that is front and center.